fs/vfs/fs_epoll.c

```diff
@@ -246,6 +246,8 @@ static int epoll_do_create(int size, int flags)
       list_add_tail(&eph->free, &epn[i].node);
     }
 
+  eph->crefs++;
```

Comment:
just a question: why `eph->crefs++;` and not `eph->crefs = 1;`?

Reply:
the result is same since eph is allocated by kmm_zalloc. I guess @DannyPlusPlus use ++ just because other place use ++/-- too.

Reply:
eph->crefs init val is zero, eph->crefs = 1 is the same as eph->crefs++